src/main/java/org/recordrobotics/charger/subsystems/Claw.java

```diff
@@ -8,32 +8,32 @@
 import com.revrobotics.CANSparkMaxLowLevel.MotorType;
 
 public class Claw extends SubsystemBase {
+
 	private static final double GEAR_RATIO = 63;
 
-	//Open
+	// Claw positions
 	public static final double CLAW_NEUTRAL = 0.0;
-	//number of turns to grab cube
 	public static final double CLAW_CUBE = 0.2;
-	//number of turns to grab cone
 	public static final double CLAW_CONE = 0.3;
 
 	private CANSparkMax _motor = new CANSparkMax(RobotMap.Claw.MOTOR_PORT, MotorType.kBrushless);
 
-	public Claw(){
+	public Claw() {
 		_motor.set(0);
 		_motor.getEncoder().setPositionConversionFactor(1 / GEAR_RATIO);
 	}
 
 
-	public void turn(double speed){
+	public void turn(double speed) {
 		_motor.set(Subsystems.limitSpeed(speed));
 	}
 
-	public double getPosition(){
+	public double getPosition() {
 		return _motor.getEncoder().getPosition();
 	}
 
 	public void resetEncoders() {
 		_motor.getEncoder().setPosition(0);
 	}
+
```

Comment:
unnecessary line

Reply:
It's pretty common to see in Java. I added it to match the newline at the start of class declaration.